app/api/v1/application_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Api::V1::ApplicationController < ActionController::API
```

Comment:
- [ ] It would be better if all our API endpoints receives JSON and responds in a JSON format as well

Reply:
Hello @addod19 ,
Thanks for your observation, could you give an example of how I could do it?